infrastructure/main.tf

```diff
@@ -31,3 +31,30 @@ data "azurerm_key_vault" "idam" {
   name                = local.vault_name
   resource_group_name = "idam-${var.env}"
 }
+
+module "redis-cache" {
```

Comment:
@henrydobson can you please confirm if that module is needed here?

Reply:
It is not.